packages/EasyBankFiles/src/Parsers/DirectEntry/Parser.php

```diff
@@ -75,155 +87,240 @@ public function getErrors(): array
-            'totalNetAmount' => $totalNetAmount === false ? null : $this->trimLeftZeros($totalNetAmount),
             'totalCreditAmount' => $totalCreditAmount === false ? null : $this->trimLeftZeros($totalCreditAmount),
             'totalDebitAmount' => $totalDebitAmount === false ? null : $this->trimLeftZeros($totalDebitAmount),
-        ]);
+            'totalNetAmount' => $totalNetAmount === false ? null : $this->trimLeftZeros($totalNetAmount),
+        ]));
     }
 
     /**
-     * Process transaction block of line.
+     * Process transaction block of line (transaction type payment = 1).
      */
-    private function processTransaction(string $line): Transaction
+    private function processTransactionTypePayment(string $line): bool
     {
-        /** @var string|false $accountName */
-        $accountName = \substr($line, 30, 32);
-        /** @var string|false $accountNumber */
-        $accountNumber = \substr($line, 8, 9);
-        /** @var string|false $amount */
-        $amount = \substr($line, 20, 10);
-        /** @var string|false $bsb */
-        $bsb = \substr($line, 1, 7);
-        /** @var string|false $lodgmentReference */
-        $lodgmentReference = \substr($line, 62, 18);
-        /** @var string|false $remitterName */
-        $remitterName = \substr($line, 96, 16);
-        /** @var string|false $traceAccountNumber */
-        $traceAccountNumber = \substr($line, 87, 9);
-        /** @var string|false $traceBsb */
-        $traceBsb = \substr($line, 80, 7);
-        /** @var string|false $txnCode */
-        $txnCode = \substr($line, 18, 2);
         /** @var string|false $withholdingTax */
         $withholdingTax = \substr($line, 112, 8);
 
-        return new Transaction([
-            'accountName' => $accountName === false ? null : \trim($accountName),
-            'accountNumber' => $accountNumber === false ? null : $accountNumber,
-            'amount' => $amount === false ? null : $this->trimLeftZeros($amount),
-            'bsb' => $bsb === false ? null : \str_replace('-', '', $bsb),
-            'indicator' => $line[17] ?? '',
-            'lodgmentReference' => $lodgmentReference === false ? null : \trim($lodgmentReference),
-            'recordType' => $line[0] ?? '',
-            'remitterName' => $remitterName === false ? null : \trim($remitterName),
-            'traceAccountNumber' => $traceAccountNumber === false ? null : $traceAccountNumber,
-            'traceBsb' => $traceBsb === false ? null : \str_replace('-', '', $traceBsb),
-            'txnCode' => $txnCode === false ? null : $txnCode,
-            'withholdingTax' => $withholdingTax === false ? null : $withholdingTax,
-        ]);
+        return $this->addTransactionToCurrentBatch(new TransactionTypePayment(\array_merge(
+            [
+                'withholdingTax' => $withholdingTax === false ? null : $this->trimLeftZeros($withholdingTax),
```

Comment:
What about renaming to `amountOfWithholdingTax` to be consistent with docs and `\EonX\EasyBankFiles\Generators\Aba\Objects\Transaction`?

Reply:
Renamed